src/client/dfuse/ops/lookup.c

```diff
@@ -321,8 +321,8 @@ dfuse_cb_lookup(fuse_req_t req, struct dfuse_inode_entry *parent,
 	rc = dfs_lookup_rel(parent->ie_dfs->dfs_ns, parent->ie_obj, name,
 			    O_RDONLY, &ie->ie_obj, NULL, &ie->ie_stat);
 	if (rc) {
-		DFUSE_TRA_DEBUG(parent, "dfs_lookup() failed: (%s)",
-				strerror(rc));
+		DFUSE_TRA_DEBUG(parent, "dfs_lookup() failed: "DF_RC"",
```

Comment:
This one is wrong, dfs is supposed to return system error numbers.

Reply:
Oh, yes, the return are positive values, will revert this change.

Reply:
This will resolve warnings in several of the dfuse ops/ files at the same time.